React/Views/RCTModalHostView.m

```diff
@@ -176,7 +168,7 @@ - (BOOL)hasAnimationType
 
 - (void)setTransparent:(BOOL)transparent
 {
-  if ([self isTransparent] != transparent) {
+  if (self.transparent != transparent) {
```

Comment:
`self.isTransparent`, no?
I am curious, how does it work? We rename the getter to `isTransparent`, but... accessing it as `self.transparent` still works. That's confusing.